academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -2,35 +2,208 @@
 
 Makes command line tool is designed to help users create and host MAKES indexes.
 
-## CreateHostResource Command
+## CreateHostResources Command
 
 Creates the MAKES index hosting resources. E.g. MAKES web host image
 
-## HostIndex Command
+```cmd
+kesm CreateHostResources --HostResourceName
+            --MakesPackage
+            --[Region]
+            --[MakesWebHost]
+```
+
+### Required Parameters
+
+`--HostResourceName`
+
+The name of the host resource.  
+
+`--MakesPackage`
+
+The base URL to a MAKES release package.
+
+### Optional Parameters
+
+`--Region`
+
+The region to deploy the host resources to. Defaults to WestUs
+
+`--MakesWebHost`
+
+The URL to MAKES web host zip file that will be used to create the MAKES virtual machine hosting image.
+
```

Comment:
I am confused on how parameters are specified. Add example of the command, something like
kesm CreateHostResources --HostResourceName someresource --MakesPackage somepackagename --MakesWebHost somewebhost